include/hpipm_d_dense_qp_ipm.h

```diff
@@ -90,7 +90,7 @@ struct d_dense_qp_ipm_arg
 	int compute_obj; // compute obj on exit
 	int split_step; // use different steps for primal and dual variables
 	int t_lam_min; // clip t and lam: 0 no, 1 in Gamma computation, 2 in solution
-	int t0_init; // 0 sqrt(mu0), 1 1.0, 2 euristic for primal feas
+	int t0_init; // initialization scheme of lambda and t slacks: 0 sqrt(mu0), 1 1.0, 2 heuristic for primal feasibility
```

Comment:
strictly speaking, this directly controls only the initialization scheme for `t`: `lam` is always initialized as `mu0/t`

Reply:
https://github.com/giaf/hpipm/commit/e57d1c171a78c9ff46e3d96224e1736b367890ec